cmd/romana.k8s

```diff
@@ -1,345 +0,0 @@
-		echo "Unable to fetch tenants using URL '${tenant_url}${tenant_path}'"
-		return
-	fi
-	read -r tenant_id _ < <(jq -r --arg name "$tenant_name" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
-
-	if ! [[ "$tenant_id" ]]; then
-		echo "Unable to find tenant id using tenant ref '$tenant_name'"
-		return
-	fi
-
-	post_data=$(printf '{"id": 0, "name": "%s"}' "$segment_name")
-	segment_path="$tenant_path/$tenant_id/segments"
-	echo "Sending request to '${tenant_url}${segment_path}': '$post_data'"
-	curl -s -X POST -H "Content-Type: application/json" --data "$post_data" "${tenant_url}${segment_path}"; printf "\n"
-}
-
-add_policy() {
-	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
-		echo "romana add-policy <tenant> <policy-file>"
-		return
-	fi
-	if (( $# != 2 )); then
-		echo "ERROR: expected 2 parameters, received $#"
-		return
-	fi
-	local tenant="$1" policy_file="$2"
-	if ! [[ -f "$policy_file" ]]; then
-		echo "Unable to find file '$policy_file'"
-		return
-	fi
-	local url=$(printf "${kube_apiserver_url}${romana_policy_path}" "$romana_policy_version" "$tenant")
-	echo "Sending POST request to '$url': '@$policy_file'"
-	curl -s -X POST -H "Content-Type: application/json" --data "@$policy_file" "$url"; printf "\n"
-}
-
-remove_policy() {
-	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
-		echo "romana remove-policy <tenant> <policy-name>"
-		return
-	fi
-	if (( $# != 2 )); then
-		echo "ERROR: expected 1 parameters, received $#"
-		return
-	fi
-	local tenant="$1" policy_name="$2"
-	local url=$(printf "${kube_apiserver_url}${romana_policy_path}/$policy_name" "$romana_policy_version" "$tenant")
-	echo "Sending DELETE request to '$url'"
-	curl -s -X DELETE "$url"; printf "\n"
-}
-
-# Show usage if no subcommand was provided
-if (( $# == 0 )); then
-	usage
-	exit 9
-fi
-
-# Undocumented options to set topology and tenant URLs
-while [[ "$1" = -* ]]; do
-	case "$1" in
-		--tenant-url)
-			tenant_url="$2"
-			shift 2
-			;;
-		--tenant-url=*)
-			tenant_url="${1:13}"
-			shift
-			;;
-		--topology-url)
-			topology_url="$2"
-			shift 2
-			;;
-		--topology-url=*)
-			topology_url="${1:15}"
-			shift
-			;;
-		*)
-			# Silently ignore unrecognized parameter
-			shift
-			;;
-	esac
-done
-
-
-# Dispatch to subcommand
-case "$1" in
-	add-host)
-		shift
-		add_host "$@"
-		;;
-	remove-host)
-		echo "$1: Not implemented yet"
-		;;
-	show-host)
-		shift
-		show_host "$@"
-		;;
-	show-hosts)
-		show_hosts
-		;;
-	create-tenant)
-		shift
-		create_tenant "$@"
-		;;
-	delete-tenant)
-		echo "$1: Not implemented yet"
-		;;
-	show-tenant)
-		shift
-		show_tenant "$@"
-		;;
-	show-tenants)
-		show_tenants
-		;;
-	add-segment)
-		shift
-		add_segment "$@"
-		;;
-	remove-segment)
-		echo "$1: Not implemented yet"
-		;;
-	add-policy)
-		shift
-		add_policy "$@"
-		;;
-	remove-policy)
-		shift
-		remove_policy "$@"
-		;;
-	help)
-		usage
-		;;
-	*)
-		echo "Unknown subcommand '$1'"
-		usage
-		exit 1
-		;;
-esac
```

Comment:
Good. Removes the temporary scripts. Is the `cmd` directory also gone?
(I can't tell via the diff)